packages/RNTester/js/components/RNTesterExampleList.js

```diff
@@ -72,16 +108,43 @@ class RowComponent extends React.PureComponent<{
                   styles.row,
                   {backgroundColor: theme.SystemBackgroundColor},
                 ]}>
-                <Text style={[styles.rowTitleText, {color: theme.LabelColor}]}>
-                  {item.module.title}
-                </Text>
-                <Text
-                  style={[
-                    styles.rowDetailText,
-                    {color: theme.SecondaryLabelColor},
-                  ]}>
-                  {item.module.description}
-                </Text>
+                <View style={styles.rowTextContent}>
+                  <Text
```

Comment:
@AnshG714 , I would suggest you make a reusable stylized `Text` component which can be used through the app without passing inline styles

Reply:
Makes sense